src/lib/release.ts

```diff
@@ -85,7 +87,11 @@ export function mapArch(): string {
         case 'arm64':
             return 'arm64';
         case 'arm64':
-                return 'arm64';
+            return 'arm64';
+        case 'ppc64':
+            return 'ppc64';
```

Comment:
based on above comment, should this be `ppc64le`?

Reply:
changed to `ppc64le`